### PR TITLE
feat: change toggle button to switch

### PR DESCRIPTION
**Pull request by asanehisa**
https://github.com/YextSolutions/pages-visual-editor-starter/pull/122
same PR ^

### CODE REVIEW

File: src/internal/puck/components/Header.tsx [hunk 1]
```diff
@@ -197,18 +192,10 @@ const ToggleEntityFields = () => {
...
-              tooltipsVisible
-                ? "ve-border-2 ve-border-[#5A58F2] ve-rounded-full"
-                : ""
-            }
-          >
-            <MessageSquareText className="sm-icon" />
-          </Button>
+          <div className="flex flex-row self-center gap-3 pl-2">
+            <Switch onCheckedChange={toggleTooltips} />
+            <p className="self-center text-sm">Entity Fields</p>
```
Comment (brian-baugher) [inline src/internal/puck/components/Header.tsx:197]: see class name comment

File: src/internal/puck/ui/switch.tsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+import * as SwitchPrimitives from "@radix-ui/react-switch";
+import { cn } from "../../utils/cn.ts";
+
+const Switch = React.forwardRef<
+  React.ElementRef<typeof SwitchPrimitives.Root>,
+  React.ComponentPropsWithoutRef<typeof SwitchPrimitives.Root>
+>(({ className, ...props }, ref) => (
+  <SwitchPrimitives.Root
+    className={cn(
+      "self-center peer inline-flex h-5 w-9 shrink-0 cursor-pointer items-center rounded-full border-2 border-transparent transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 focus-visible:ring-offset-background disabled:cursor-not-allowed disabled:opacity-50 data-[state=checked]:bg-primary data-[state=unchecked]:bg-input",
```
Comment (brian-baugher) [inline src/internal/puck/ui/switch.tsx:11]: Tailwind classes in the ve library are prefixed with "ve-" to isolate their styles. So all of these class names have to be prefixed like "ve-items-center". This is true wherever you're using tailwind classes in this lib.

File: src/internal/puck/ui/switch.tsx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+      className
+    )}
+    {...props}
+    ref={ref}
+  >
+    <SwitchPrimitives.Thumb
+      className={cn(
+        "self-center pointer-events-none block h-4 w-4 rounded-full bg-background shadow-lg ring-0 transition-transform data-[state=checked]:translate-x-4 data-[state=unchecked]:translate-x-0"
```
Comment (brian-baugher) [inline src/internal/puck/ui/switch.tsx:19]: same

**Review by brian-baugher — CHANGES_REQUESTED** [review]

File: src/internal/puck/ui/switch.tsx (hunk 2 above)
Comment (asanehisa) [reply src/internal/puck/ui/switch.tsx:11]: Gotcha. I think there are some libraries where u can have "ve-" automatically added to all the tailwind classes too on build or smth. I'll fix tho!

**Review by brian-baugher — APPROVED** [review]

**Pull request merged**